src/bench.c

```diff
@@ -309,15 +339,14 @@ char *benchmark_format(struct fmt_main *format, int salts,
 	static int wait_salts = 0;
 	char *where;
 	struct fmt_tests *current;
-	int cond;
+	int pass;
 #if OS_TIMER
 	struct itimerval it;
 #endif
-	clock_t start_real, end_real;
-	clock_t start_virtual, end_virtual;
 #if !defined (__MINGW32__) && !defined (_MSC_VER)
 	struct tms buf;
 #endif
+	clock_t start_real, start_virtual, end_real, end_virtual;
```

Comment:
No-op changes like the above may look strange or irrelevant but I simply took some chances to get Jumbo's code closer to core, for things like `git diff origin/master bench.c` to be as clean as possible.